routing/result_interpretation.go

```diff
@@ -267,6 +267,21 @@ func (i *interpretedResult) processPaymentOutcomeFinal(
 		// destination correctly. Continue the payment process.
 		i.successPairRange(route, 0, n-1)
 
+	// We do not expect to receive an invalid blinding error from the final
+	// node in the route. This could erroneously happen in the following
+	// cases:
+	// 1. Unblinded node: misuses the error code.
+	// 2. A receiving introduction node: erroneously sends the error code,
+	//    as the spec indicates that receiving introduction nodes should
+	//    use regular errors.
+	//
+	// Note that we expect the case where this error is sent from a node
```

Comment:
Suggestion:

```
Note that we expect the case where the blinded path has more hops than just the introduction node and we receive a final hop error from a node after the introduction node to be handeld elsewhere as ....
```

Reply:
I don't find this clearer than the existing comment, but happy to update if other reviewers feel differently!